website/content/docs/troubleshoot/faq.mdx

```diff
@@ -6,7 +6,24 @@ sidebar_title: FAQ
 
 # Frequently Asked Questions
 
-## Q: What is Checkpoint? / Does Consul call home?
+## Consul on Kubernetes
+
+### Q: Can I upgrade directly to a specific Helm chart version or should I upgrade one patch release at a time?
+
+It is safe to upgrade directly to a specific version. Be sure to read the release notes for all versions you're upgrading
+through and look for any breaking changes.
+
+### Q: Can I upgrade in place or should I spin up a new Kubernetes cluster?
+
+It is always safer to spin up a new Kubernetes cluster but that is not an
+option for most teams. Consul supports [upgrading in place](/docs/k8s/upgrade).
+
+Non-production environments should be upgraded first. If upgrading
+a Consul version, Consul data should be backed up.
```

Comment:
If there are backup and restore docs, can we link them here?

Reply:
👍 linked to https://learn.hashicorp.com/tutorials/consul/kubernetes-disaster-recovery